projects/aws/upgrader/upgrade/cniplugins_upgrade_test.go

```diff
@@ -0,0 +1,21 @@
+package upgrade_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+)
+
+const (
+	cmpDir = "/foo/binaries"
+)
+
+func TestCniPluginsUpgrade(t *testing.T) {
```

Comment:
Testing error cases?

Reply:
All os/sys calls could return an error. And we aren't having any special error handling logic in the main flow. I don't see a value in adding/covering tests with an error scenario for each of the os calls. That would make the test file unreasonably big with little added value. Goal or idea here was to cover the happy flow to see the sequence of calls being made were validated and as expected.